src/components/network/network-panel.js

```diff
@@ -135,8 +148,21 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.source.label);
```

Comment:
Here please retrieve label from the node objects. The function setEdgeInfo_ can accept edge as network.Edge.